src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    add: function(arg) {
```

Comment:
please, before make PR, make sure that you checked all points from checklist

https://github.com/mate-academy/js_advanced_calculator/blob/master/checklist.md

Reply:
did you open checklist?
please, make sure that you fixed all comments, before re-requesting review
![image](https://user-images.githubusercontent.com/59209337/209776883-6b43217e-c3ef-4f5b-8d9b-b0c1e3ee2502.png)

![image](https://user-images.githubusercontent.com/59209337/209776861-a26c538d-ed0f-4062-9c58-5c72b13a6f5b.png)